### PR TITLE
Various corrections given a complete review of the doc

### PR DESCRIPTION
**Pull request by huntc**

### CODE REVIEW

File: src/main/play-doc/api/BundleAPI.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ Content-Type: application/json
 ]
 ```
 
-Where `10.11.23.22:7001` and `10.11.23.22:7002` are the host and port of the service which is currently running.
+Where `10.11.23.22:7001` and `10.11.23.22:7002` are the host and host port of the service which is currently running.
```
Comment (fsat) [inline src/main/play-doc/api/BundleAPI.md:84]: @huntc - `host port` is the bound port of the service which is currently running, isn't it?

File: src/main/play-doc/api/BundleAPI.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ Cache-Control: max-age={max-age}
 
 Field        | Description
 -------------|------------
-location-url | The location of the requested service including any trailing parts to the path requested e.g. `/customers/123` would result in `http://10.0.1.22:8080/customers/123` supposing that ConductR's proxy service's address is `10.0.1.22`, and the endpoint service port is `8080`. If the protocol for the service was TCP then that will be reflected in the returned location's protocol field. For example looking up `/jms` may yield `tcp://10.0.1.22:61616` supposing that the Active/MQ JMS broker's service is offered on port 61616.
+location-url | The location of the requested service including any trailing parts to the path requested e.g. `/customers/123` would result in `http://10.0.1.22:10121/customers/123` supposing that service's host address is `10.0.1.22`, and the endpoint service host port is `10121`. If the protocol for the service was TCP then that will be reflected in the returned location's protocol field. For example looking up `/jms` may yield `tcp://10.0.1.22:10121`.
```
Comment (fsat) [inline src/main/play-doc/api/BundleAPI.md:44]: @huntc - I know this statement has been around in the existing doc:

> For example looking up `/jms` may yield `tcp://10.0.1.22:10121`

Is this statement correct? If I were to lookup `/jms` through the service locator, will it return HTTP `307` with `tcp://10.0.1.22:10121` in the `Location` field?

File: src/main/play-doc/api/BundleAPI.md (hunk 1 above)
Comment (huntc) [inline src/main/play-doc/api/BundleAPI.md:84]: The host port is mapped to the bind port. External services always use the host port. They are generally one of the same, unless you're using a container, in which case the bind port can be whatever you want it to be.

File: src/main/play-doc/api/BundleAPI.md (hunk 2 above)
Comment (huntc) [inline src/main/play-doc/api/BundleAPI.md:44]: That is correct.

**Comment by fsat** [comment]
LGTM

File: src/main/play-doc/api/BundleAPI.md (hunk 2 above)
Comment (huntc) [inline src/main/play-doc/api/BundleAPI.md:44]: Actually, I think we will just return the protocol of the original request... that would be a bug technically...

**Comment by huntc** [comment]
#168 takes care of master

**Pull request merged**